extensions/adobe/experience/offer-management/offer-metadata.schema.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
This one makes no sense!  Every XDM is metadata.  What is this?!?

Reply:
Removed